google/__init__.py

```diff
@@ -28,4 +28,10 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # pylint: disable=missing-docstring
-__import__('pkg_resources').declare_namespace(__name__)  # pragma: no cover
+from __future__ import absolute_import
```

Comment:
If we want to handle that case here, then we should probably also update the GAPIC templates as well ([here](https://github.com/googleapis/toolkit/blob/master/src/main/resources/com/google/api/codegen/py/namespace__init__.py.snip), [here](https://github.com/googleapis/toolkit/blob/master/src/main/resources/com/google/api/codegen/py/namespace__init__.py.snip)).

It's unlikely anyone's using GAX without GAPIC. In any case, it doesn't seem harmful to do this here.